### PR TITLE
Form section

### PR DESCRIPTION
**Pull request by rivasbolinga**
Adding last part of mobile version of my portfolio: Contact-form section and footer  with social media logos,

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -155,5 +163,54 @@ <h5 class="skills-title">Skills</h5>
...
     </section>
+
+     <!-- Form section -->
+
+    <section class="form-section">
+      <div class="form-header">
+        <h1 class="form-title">Get started</h1>
+        <p class="form-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      </div>
+        <form action="https://formspree.io/html/" method="POST" class="form">
```
Comment (ridwanediallo) [inline portfolio.html:174]: - [ ] Hello @rivasbolinga , good job so far 👏. But the form submission is still not working.
 
Screenshot from 2022-11-01 23-09-05

- [x] What you need first is to create your account at [formspree](https://formspree.io/html/) and then generate the `form_id` by following the steps. 
when you got the `id`, your `action` should be similar to something like this `action="https://formspree.io/f/hgkd1254us"`
- [x] Lastly, please make that a user can successfully submit a `form`. After that is done you can proudly submit it for review. 🤗

Comment (ridwanediallo) [reply portfolio.html:174]: ## How to get the `action` value ?
[Demo video](https://user-images.githubusercontent.com/89228743/199359877-b51846d3-16b3-488d-92c2-a992ecee2b21.webm)

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @rivasbolinga , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.html [hunk 2]
```diff
@@ -155,5 +163,54 @@ <h5 class="skills-title">Skills</h5>
...
+     <!-- Form section -->
+
+    <section class="form-section">
+      <div class="form-header">
+        <h1 class="form-title">Get started</h1>
+        <p class="form-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      </div>
+        <form action="https://formspree.io/f/mvoyvvge" method="POST" class="form">
+          <div class="form-field">
+          <input name="name" type="text" placeholder="Your name" maxlength="30" required value="Andrea">
```
Comment (GiuseppeTG) [inline portfolio.html:176]: - [OPTIONAL] I highly suggest that you remove de value from the inputs, it doesn't let the placeholder do its job 🤓. Also, the user has to erase that value before typing his credentials 👍

**Review by GiuseppeTG — APPROVED** [review]
Hi @rivasbolinga  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 : 
Congratulations! 🎉 

Alt Text

## Highlights
✅  Good documentation
✅  Changes implemented correctly

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**